### DISCUSSION TITLE
Links on images with hover

### DISCUSSION THREAD

**Discussion by Tychodewaard** [vanjarosoftware/Vanjaro.Platform#1514] [Q&A]
On https://tjeps10.tip15.40fingers.eu/Overons I have several icons. On hover they alternate with another version of the icon.
If I want to have this icon a link, i can add a link to the page and drag the image within the link. But as soon as I do that, the on hover effect of the image disappears.

What I want, is that possible (without custome code)?
If so, how?

**Comment by Gaganbird** [comment]
I was unable to duplicate the issue with the nightly build. Please try with the official release (coming soon).

**Reply by eswanzey** [reply]
The pre-dawn time: "The pre-release experience is no longer available. We're preparing for official release."

What makes me very happy is confirmation from @Gaganbird that the nightly build will comprise the release version. Otherwise,  if I look at the v1.0.6 milestone, it is 19 days stale.
